### PR TITLE
Updates

### PR DESCRIPTION
**Pull request by fcastilloec**
Update multiple modules and some fixes, follow a list of some of the most important ones
- Update `mpris` and switch to using `dbus-native` module over `dbus`
- Use `lvh.me` for the player window
- Update to `electron@v3.0.7`
- Fix Windows' main window resize bug while standardizing all OSs window sizes

### To-do (maybe in a different PR):

- [x] Disable close button on `player` window for all OSs
- [x] Remove user dialog when attempting to close `player` window on macOS

### CODE REVIEW

File: windows/index.js [hunk 1]
```diff
@@ -23,33 +22,39 @@ let player;
...
   logger.info('Starting Headset');
   const mainWindowState = windowStateKeeper();
 
   win = new BrowserWindow({
     x: mainWindowState.x,
     y: mainWindowState.y,
-    width: 381,
-    height: 696,
+    width: 375,
+    height: 667,
```
Comment (danielravina) [inline windows/index.js]: why is that needed?

File: darwin/index.js [hunk 2]
```diff
@@ -88,7 +92,7 @@ const start = () => {
     }, 2000);
 
     if (isDev) {
-      player.loadURL('http://127.0.0.1:3001');
+      player.loadURL('http://lvh.me:3001');
```
Comment (m1guelpf) [inline darwin/index.js]: Why the switch?

Comment (danielravina) [reply darwin/index.js]: many yt videos are blocked when request is coming from 127.0.0.1

Comment (fcastilloec) [reply darwin/index.js]: The Youtube API seems to not work properly if you load the player from `127.0.0.1`, some videos were skipped without giving any errors. `lvh.me` is a dummy website the redirects to `127.0.0.1`, this way Youtube doesn't complain :)

**Comment by danielravina** [comment]
Thanks!
